core/src/main/java/io/grpc/ServerInterceptors.java

```diff
@@ -179,101 +173,96 @@ public InputStream parse(final InputStream stream) {
   public static <T> ServerServiceDefinition useMarshalledMessages(
       final ServerServiceDefinition serviceDef,
       final MethodDescriptor.Marshaller<T> marshaller) {
-    final ServerServiceDefinition.Builder serviceBuilder = ServerServiceDefinition
-        .builder(serviceDef.getName());
-    for (final ServerMethodDefinition<?, ?> definition : serviceDef.getMethods()) {
-      final MethodDescriptor<?, ?> originalMethodDescriptor = definition.getMethodDescriptor();
-      final MethodDescriptor<T, T> wrappedMethodDescriptor = MethodDescriptor
-          .create(originalMethodDescriptor.getType(),
-              originalMethodDescriptor.getFullMethodName(),
+    List<MethodDescriptor> wrappedMethods = new ArrayList<MethodDescriptor>(
```

Comment:
We have -Xlint:rawtypes enabled, and it does appear like the build [warned for this usage](https://travis-ci.org/grpc/grpc-java/jobs/131273766#L305) (along with lots of others).